src/tester.rs

```diff
@@ -144,6 +150,28 @@ impl<'a> LangTester<'a> {
         self
     }
 
+    /// If set to `true`, this reads arguments from `std::env::args()` and interprets them in the
+    /// same way as normal cargo test files. For example if you have test "ab" and "cd" but only want
```

Comment:
typo, should this be "If you have **tests** "ab" and..." ?

Reply:
Oops!